src/main/resources/api/service_openapi.yaml

```diff
@@ -582,11 +582,7 @@ components:
     ReferenceTypeEnum:
       type: string
       description: The type of a reference, if an uncontrolled resource
-      # TODO: swagger-codegen always looks for and removes a common prefix for
-      # enum values, which breaks enums that only have a single value.
-      # PLACEHOLDER_VALUE should be removed when we have an actual second value
-      # to use.
-      enum: ['DATA_REPO_SNAPSHOT', 'PLACEHOLDER_VALUE'] #eventually include GCS bucket, etc.
+      enum: ['DATA_REPO_SNAPSHOT', 'GOOGLE_BUCKET', 'BIG_QUERY_DATASET']
```

Comment:
glad to have `PLACEHOLDER_VALUE` out of here!

Reply:
Same, that truncation is still one of the strangest opinions I've seen swagger take.